e2e/__tests__/__snapshots__/showConfig.test.ts.snap

```diff
@@ -72,7 +72,8 @@ exports[`--showConfig outputs config info and exits 1`] = `
         ]
       ],
       "transformIgnorePatterns": [
-        "/node_modules/"
+        "/node_modules/",
+        "\\\\.pnp\\\\.c?js$"
```

Comment:
Should we do `.pnp.*` to cover `.json` as well? Or doesn't that file exist anymore?

/cc @arcanis probably knows 😃

Reply:
There's `.pnp.data.json` under certain configurations, so better just use a wildcard

Reply:
There's `.pnp.data.json` under certain configurations, so better just use a wildcard

Reply:
Do JSON files really can go through transformers? I'll do this, but afraid this can brake something like `src/.pnp.foo/bar.js`

Reply:
Yup: https://github.com/facebook/jest/pull/8278

Reply:
`"\\.pnp\\..+"` or more strict `"\\.pnp\\.c?js(on)?$"`?

Reply:
I'd go with `\\.pnp\\.[^\\\/]+`, and refine if it causes problems in practice.